terraform/modules/application/alarms.tf

```diff
@@ -1,6 +1,6 @@
-resource "aws_cloudwatch_metric_alarm" "app_cpu" {
+resource "aws_cloudwatch_metric_alarm" "app_average_cpu" {
   alarm_actions             = [var.sns_topic_arn]
-  alarm_name                = "${var.prefix}-app-cpu"
+  alarm_name                = "${var.prefix}-app-average-cpu"
```

Comment:
I think the name change is wanted, which forces a new resource anyway hence the lack of moved blocks.